learn-css-grid-by-building-a-magazine/index04.css

```diff
@@ -0,0 +1,8 @@
+.magazine {
+  display: grid;
+  /* To declare the number of columns i.e. vertical divisions a grid has, we use the
+  grid-template-columns property. We use a unit called fr that represents a fraction of the free space.
+  For example: grid-template-columns: 1fr 3fr; would divide the grid into 1+3=4 parts, the first column
+  would occupt 25% of the space and the second one would occupy 75%
```

Comment:
`oocupt` -> `occupy`

Reply:
I've resolved this locally, would be pushed up with next update